deployment/README-docker.md

```diff
@@ -85,6 +85,46 @@ e.g.
 ```
 sudo /etc/init.d/nginx restart
 ```
+### Run  project in development environment
+
+Firstly you need to be in the project **deployment** directory.
+
+```
+cd /path/to/feti/deployment/
+``` 
+
+Then after you may need to start the containers independently with the commands below, if your docker setup requires sudo privileges do include it.
```

Comment:
sudo is not needed if your user is in the dockers group. So lave out the sudo in the below commands.

Reply:
Alright @cchristelis